docstr_badge.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
With the pass/fail badge plan, I believe this file isn't being referenced any more. Please delete it, and then we're good for merge.

Reply:
Oh, my bad, I have got it deleted now.